dev/Lights/ApiTests/Lights_ApiTests/LightConfigurationTests.cs

```diff
@@ -230,7 +230,7 @@ void VerifyLights(CoreApplicationView whichView, bool forceLightAttachDuringLayo
                 }
                 else
                 {
-                    MUXControlsTestApp.App.TestContentRoot = mySPRoot;
+                    Content = mySPRoot;
```

Comment:
Do you need to call Content.UpdateLayout() somewhere?

Reply:
I did not update every test to use UpdateLayout() instead of IdleSynchronizer yet. I think these more complicated ones might be easier on a subsequent pass.

---
In reply to: [358512055](https://github.com/microsoft/microsoft-ui-xaml/pull/1760#discussion_r358512055) [](ancestors = 358512055)